src/templates/mht_document.tpl

```diff
@@ -0,0 +1,22 @@
+From: "Generated by html2docx lib"
+Subject: Html To Docx Document
+Date: Thu, 6 Aug 2015 02:49:38 -0700
```

Comment:
Deleting those 'email' fields doesn't seem to cause any issue - files are opened correctly without errors (tested on MS Word 2010 on Win and MS Word 2011 on Mac and Outlook Online), but I'd still keep them. All browsers with MHT export support (IE, Opera, Chrome) keep those fields when saving a web page.